hack/storage-version-migration.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+# Copyright The Shipwright Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -euo pipefail
+
+if ! hash jq >/dev/null 2>&1 ; then
+  echo "[ERROR] jq is not installed"
+  exit 1
+fi
+
+# Delete old job for storage version migration
+kubectl -n shipwright-build delete job --selector app=storage-version-migration-shipwright --wait=true
+
+# create new job for storage version migration
+cat <<EOF | kubectl create -f -
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: storage-version-migration-shipwright-
+  labels:
+    app: storage-version-migration-shipwright
+    app.kubernetes.io/component: storage-version-migration-job
+    app.kubernetes.io/name: shipwright-build
+    app.kubernetes.io/version: devel
```

Comment:
Not needed.

Reply:
@SaschaSchwarze0  Do you mean, just this label `app.kubernetes.io/version: devel` or all the labels ?